client/src/components/questions-answers/individual-question/Index.jsx

```diff
@@ -1 +1,86 @@
-import React from 'react';
\ No newline at end of file
+import React, { useEffect, useState }from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Container, Row, Col } from 'react-bootstrap';
+import { bindActionCreators } from 'redux';
+import { actionCreators } from '../../../redux/index.js';
+import AnswerList from '/Users/gitayier/Desktop/Catbird/client/src/components/questions-answers/individual-question/AnswerList.jsx';
```

Comment:
Can you change this to a relative location? Can't render that component unless it's relative. Should just be './AnswerList.jsx'